src/pages/docs/w3up-client.md

```diff
@@ -90,7 +86,17 @@ If your agent has no other spaces, saving the space will set it as the "current
 await client.setCurrentSpace(space.did())
 ```
 
-ℹ️ Note: creating a space and provisioning it needs to happen only **once**!
+One last thing - now that you've saved your space locally, it's a good idea to setup recovery, so that when you move to a different device you can still access your space: 
```

Comment:
```suggestion
One last thing - now that you've saved your space locally, it's a good idea to setup recovery via [web3.storage Account](https://github.com/web3-storage/specs/blob/main/w3-account.md), so that when you move to a different device you can still administer your space as long as you can log in to your web3.storage Account: 
```

just an idea here to explain *what kind of recovery*. IMO without this info, it's not possible to understand what's going on and how to reason about how doing this affects who can write/admin the space.